packages/client/test/integration/fullsync.spec.ts

```diff
@@ -8,8 +8,9 @@ tape('[Integration:FullSync]', async (t) => {
     const [localServer, localService] = await setup({ location: '127.0.0.1', height: 0 })
     await localService.synchronizer.stop()
     await localServer.discover('remotePeer1', '127.0.0.2')
+    // await localService.synchronizer.sync()
```

Comment:
```suggestion
    await localService.synchronizer.sync()
```

Reply:
Sorry, I just forgot to remove that.  The `localService` calls that `sync` function automatically so we shouldn't need it here.  This is leftover dead code from when I was triaging the integration tests.

Reply:
Or actually, 3 lines down :facepalm: